tests/acceptance/cli/test_deploy_commands.py

```diff
@@ -30,6 +34,14 @@
 ALTERNATE_REGISTRY_FILEPATH_2 = '/tmp/nucypher-test-registry-alternate-2.json'
 
 
+@pytest.fixture(autouse=True)
+def monkeypatch_confirmations(testerchain, monkeypatch):
+    def monkey_block_until_enough_confirmations(client, transaction_hash, timeout, confirmations):
```

Comment:
Does it actually block until enough confirmations? `confirmations` is unused

Reply:
Ah, my bad, it's supposed to ignore the confirmations

Reply:
:+1: - Good to indicate for future reviewers though - there is another test `test_block_confirmations` that checks for that behavior, so @derekpierre and I built this as a mock.